rtbkit/common/augmentor_interface.cc

```diff
@@ -0,0 +1,51 @@
+#include "rtbkit/common/messages.h"
+#include "augmentor_interface.h"
+
+using namespace Datacratic;
+using namespace RTBKIT;
+
+AugmentorInterface::AugmentorInterface(ServiceBase & parent,
+                                 std::string const & serviceName) :
+    ServiceBase(serviceName, parent),
+    router(nullptr){
```

Comment:
You should not need a reference to the router in the interface, so I think you can remove it.